### PR TITLE
Odk2

### PR DESCRIPTION
**Pull request by maprehensive**
Addition of the TSC2 folder and readme doc
Adding TSC2 members to TECHNICAL-STEERING-COMMITTEE.MD

### CODE REVIEW

File: TSC2/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+- [ODK Roadmap](https://github.com/opendatakit/roadmap/projects/2)
+- [Uberconference Room](https://www.uberconference.com/opendatakit)
+- [TSC Forum Category](https://forum.opendatakit.org/c/governance/tsc)
+- [TSC Forum Tag](https://forum.opendatakit.org/tags/tsc)
+- ODK 2 Project Pages: [Tables](https://github.com/opendatakit/tables), [Survey](https://github.com/opendatakit/survey), [Services](https://github.com/opendatakit/services), [Documentation](https://github.com/opendatakit/docs), [Application Designer](https://github.com/opendatakit/app-designer), [Suitcase](https://github.com/opendatakit/suitcase), [Sync-Endpoint](https://github.com/opendatakit/sync-endpoint)
+- [TSC Governance Guidelines](https://github.com/opendatakit/governance/blob/master/TECHNICAL-STEERING-COMMITTEE.md)
+- [Public Governance Page with TSC Members & Photos](https://opendatakit.org/community/governance/)
+- [Meeting Facilitator Checklist](#meeting-facilitator-checklist)
+
+For more details on the TSC's responsibilities, see the [TSC governance](https://github.com/lognaturel/governance/blob/master/TECHNICAL-STEERING-COMMITTEE.md) approved on Nov 7, 2017.
```
Comment (jbeorse) [inline TSC2/README.md]: I think we should update this note to distinguish that this link is to TSC 1, and that we are using their same responsibilities for now.

The "approved on" date should probably be updated too.

**Comment by maprehensive** [comment]
@wbrunette Thanks for clarifying. Would you recommend reverting to the original wording?

**Comment by wbrunette** [comment]
Yes I think reverting to the original wording is fine solution.

**Comment by maprehensive** [comment]
@wbrunette Have changed back to the original wording. I did this another commit, which might not be the best way, but I wasn't confident with reverting the commit. Hope is all good. 
Thanks!

**Pull request merged**